sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -75,6 +75,20 @@
     }
 }
 
+# gpu count -> [vm types]
+_L4_INSTANCE_TYPE_DICT = {
```

Comment:
Actually, can we merge `_A100_INSTANCE_TYPE_DICTS` and `_L4_INSTANCE_TYPE_DICT` and rename it to `_ACC_INSTANCE_TYPE_DICTS`? so we can simplify the code quite a bit